sec_certs_page/templates/fips/search/merged.html.jinja2

```diff
@@ -0,0 +1,334 @@
+{% extends "common/base.html.jinja2" %}
+{% set crumbs = True %}
+{% set search = True %}
+{% if searchType == "byName" %}
```

Comment:
Snake-case instead. Also, just import everything, why guard the imports like this?